src/app/doctor/doctor.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "../person/person.hpp"
+
+#include <set>
+
+class Visit;
+
+class Doctor : public Person, public std::enable_shared_from_this<Doctor>
+{
+    std::set<std::shared_ptr<Visit>> visitAssociation_;
+
+  public:
+    Doctor(std::string firstName, std::string lastName, std::string pesel);
+
+    void addAssociation(std::shared_ptr<Visit> visit);
+    std::set<std::shared_ptr<Visit>> getVisitAssociations();
```

Comment:
Same here, it could be called getVisits()

Reply:
Personally, I think it's good practice to note that we're referring to an association, not just a container to the pointers of an objects.